packages/Heading/Heading.jsx

```diff
@@ -81,6 +81,9 @@ export const StyledHeading = styled.h1(wordBreak, ({ level, invert }) => {
   return {
     color,
     ...HeadingLevels[`${level}`],
+    '& > *': {
```

Comment:
Should we target `span` specifically? I wonder if this selector will affect superscripts.

Reply:
I couldn't think of other scenarios but `span` but thought there could be some edge-cases. Since most of other edge-cases will go against accessibility (like nested `div` for example) I think we should target `span` specifically.

Reply:
This is done.